modal/image.py

```diff
@@ -274,6 +274,8 @@ def pip_install_from_requirements(
     def poetry_install_from_file(
         self,
         poetry_pyproject_toml: str,
+        poetry_lockfile: Optional[str] = None,
+        ignore_lockfile=False,
```

Comment:
Nit: Perhaps add some comments to the args, like the lockfile being automatically inferred and required by default?

Reply:
Yeah, good point, I'll add that!